src/main/java/space/obminyashka/items_exchange/exception/EmailTokenExpiredException.java

```diff
@@ -0,0 +1,11 @@
+package space.obminyashka.items_exchange.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.BAD_REQUEST)
```

Comment:
Not Acceptable response would fit that exception cause much better because 400 would be thrown by Spring when UUID format is not valid.